spec/requests/portfolio_items_spec.rb

```diff
@@ -104,6 +104,33 @@
     end
   end
 
+  describe 'GET /portfolio_items/{portfolio_item_id}/undelete' do
+    let(:undelete) { get "#{api}/portfolio_items/#{portfolio_item_id}/undelete", :headers => default_headers }
+
+    before do
```

Comment:
I think this before should be moved into the context where something needs to be deleted, it's weird to do this setup for the spec where something "has not been discarded" even though it has, it's just been undeleted afterwards as well. It's just extra setup that doesn't need to happen.

Reply:
Reworked these tests, please have a look again.